mobile/src/main/java/com/wedriveu/mobile/model/UserLocation.java

```diff
@@ -0,0 +1,73 @@
+package com.wedriveu.mobile.model;
+
+/**
+ * <p>
+ *     Describes the user location, composed by the position where the user is, and the destination position the user chooses.
+ * </p>
+ *
+ */
+public class UserLocation {
+
+    private Double GPSLatitude;
```

Comment:
Please follow camelCase for Java fields and variables.
Also, is this really a `UserLocation`? Maybe it's a `SchedulingLocation` that contains two `android.Location` class (user and destination) or `Place` class if exist that are filled with these `double`.